js/widgets.js

```diff
@@ -88,7 +153,7 @@ function easeCopyPasting (content) {
 /**
  * Display/hide the integration box for embeds.
  */
-function handleIntegration (event) {
+function toggleIntegration (event) {
   event.preventDefault()
   const element = event.target
```

Comment:
This whole sequence of DOM traversal tightly couples the code and the DOM order. A good pattern is to use `js-` prefixed classnames to make refactoring easier.